tests/transactional/install_updates.pm

```diff
@@ -34,12 +46,12 @@ sub run {
     }
 
     # First we update the system
-    fully_patch_system(trup_call_timeout => 1800);
+    update_system;
 
-    # Now we add the incident repositories and do a zypper patch
+    # Now we add the incident repositories and update the system
     add_test_repositories;
     record_info('Updates', script_output('zypper lu'));
-    trup_call('patch', timeout => 300, proceed_on_failure => 1) unless get_var('DISABLE_UPDATE_WITH_PATCH');
+    update_system unless get_var('DISABLE_UPDATE_WITH_PATCH');
```

Comment:
Why do we need two separate settings? We have now `TRANSACTIONAL_UPDATE_UP` and `DISABLE_UPDATE_WITH_PATCH` and they are somehow coupled. Do we really have scenarios in which we are running `zypper patch` and then need a switch to run `zypper up`? :thinking:

Reply:
`DISABLE_UPDATE_WITH_PATCH` was there before, not sure where it is used, I left it there not to break any current functionality.